libs/damap/src/assets/styles.scss

```diff
@@ -1,6 +1,8 @@
 html,
 body {
   height: 100%;
+  zoom: 90%;
```

Comment:
I like how the dashboard looks zoomed, but imo working on dmps is better with 100% or 95% zoom. What do you say?

Also firefox didnt support zoom until very recently, so older browsers wont see the zoom effect. Although i dont think this should be a back problem, just wanted to let you know.